### DISCUSSION TITLE
Text in bold cannot be parsed

### DISCUSSION THREAD

**Discussion by zhaixiuyan** [meteorlxy/vue-showdown#38] [Q&A] [answered]
I use it as a VUE component and the text in bold cannot be parsed.
Here is my code：
`<VueShowdown markdown="** Test bold：**" flavor="github" :options="{ emoji: true }" />`

The above code resolves to：
`**Test bold：**`

How can I fix it？Looking forward to your answer！

**Comment by meteorlxy** [comment, marked as answer]
Because it's invalid markdown bold syntax. You should remove the space after the asterisk.

Test it directly on github:

### Input

```md
** Test bold：**

**Test bold：**
```

### Output


** Test bold：**

**Test bold：**
